src/ui/hooks/comments/useAddComment.tsx

```diff
@@ -32,7 +32,7 @@ export default function useAddComment() {
     return addComment({
       variables: {
         input: {
-          ...omit(comment, ["id", "createdAt", "updatedAt", "replies", "user"]),
+          ...omit(comment, ["id", "createdAt", "updatedAt", "isUnpublished", "replies", "user"]),
```

Comment:
For now, this is a local-only field. Even once it's a field Apollo supports, we'll probably still want to filter it out (just like we filter out the id and dates) since Apollo will determine the correct value for comments that are successfully published.